samples/app/src/androidTest/java/com/sourcepoint/app/v6/MainActivityKotlinTest.kt

```diff
@@ -113,14 +114,13 @@ class MainActivityKotlinTest {
         +(CampaignType.CCPA)
     }
 
-    private val spConfBohdan = config {
+    private val spConfigMobileBohdanTestGdpr = config {
```

Comment:
please don't use personal name 😅

Reply:
I know, but the name of the property has my name in it :D

Reply:
I saw it.. Creating a property requires time so I didn’t feel to ask it